src/components/Page/components/Header/tests/Header.test.tsx

```diff
@@ -34,7 +34,7 @@ describe('<Header />', () => {
       const warnSpy = jest.spyOn(console, 'warn');
       mountWithAppProvider(<Header {...mockProps} icon="foo" />);
 
-      expect(warnSpy).toHaveBeenCalledTimes(1);
+      expect(warnSpy).toHaveBeenCalled();
```

Comment:
`React.StrictMode` is causing a second render. I'm not sure why, but I don't think it's a huge deal.

Reply:
It does that intentionally to try and make it side effects obvious as per https://reactjs.org/docs/strict-mode.html#detecting-unexpected-side-effects

Reply:
Neat 😍 

<img width="992" alt="Screen Shot 2019-06-20 at 2 22 55 AM" src="https://user-images.githubusercontent.com/24610840/59825562-5bbf6b80-9302-11e9-87a1-f8f3f03e651b.png">